src/app/my-messages/community-role-changed-notification.ts

```diff
@@ -0,0 +1,10 @@
+import { AbstractNotification } from './abstract-notification';
+import { Community } from '../communities/community';
+
+export class CommunityRoleChangedNotification extends AbstractNotification {
+  community: Community;
+
+  getTypeText(): string {
+    return 'Changes community-role';
```

Comment:
I suggest to have a message like "The community role changed."

Reply:
thank you. done